duckduckgo/Android
# Feature/share url

Task/Issue URL: https://app.asana.com/0/547792610048271/364465928695405
Tech Design URL:
CC:

**Description**:
Adds support for sharing the current page or for sharing by long pressing on a link

**Steps to test this PR**:
1. Launch the app; ensure you **can't** share the landing page
1. Search, and ensure you **can** share the SERP
1. Long press on a link, and ensure you **can** use the context menu to share the link
1. Visit a 3rd party webpage, and use the overflow menu and ensure you **can** share that page correctly


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
